src/vstest.console/Processors/CollectArgumentProcessor.cs

```diff
@@ -229,17 +229,15 @@ internal static class CoverletConstants
             /// </summary>
             public const string CoverletDataCollectorFriendlyName = "XPlat Code Coverage";
 
-            /// TODO : Finalize Name
             /// <summary>
             /// Coverlet inproc data collector assembly qualified name
             /// </summary>
-            public const string CoverletDataCollectorAssemblyQualifiedName = "Microsoft.TestPlatform.Extensions.CoverletCoverageDataCollector.CoverletCoverageDataCollector, Microsoft.TestPlatform.Extensions.CoverletCoverageDataCollector, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a";
+            public const string CoverletDataCollectorAssemblyQualifiedName = "Coverlet.Collector.DataCollection.CoverletInProcDataCollector, coverlet.collector, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null";
 
-            /// TODO : Finalize codebase
             /// <summary>
             /// Coverlet inproc data collector codebase
             /// </summary>
-            public const string CoverletDataCollectorCodebase = "coverletinprocdatacollector.dll";
+            public const string CoverletDataCollectorCodebase = "coverlet.collector.dll";
```

Comment:
What is the name of the out proc one ?

Reply:
There is only one dll coverlet.collector. The inproc datacollector is inside this in a separate folder. 
Link: [https://github.com/tonerdo/coverlet/tree/master/src/coverlet.collector](https://github.com/tonerdo/coverlet/tree/master/src/coverlet.collector)

Reply:
The publicKeyToken can be removed if null

Reply:
Guys if I clone repo and do some update on public key...is there a guide that explain how to run my local "dotnet test" driver?
If so I could test on my local if there are issue with collector load and do a PR directly to this repo.